java/rocksjni/portal.h

```diff
@@ -7894,6 +7894,40 @@ class SanityLevelJni {
   }
 };
 
+// The portal class for org.rocksdb.PrepopulateBlobCache
+class PrepopulateBlobCacheJni {
+ public:
+  // Returns the equivalent org.rocksdb.PrepopulateBlobCache for the provided
+  // C++ ROCKSDB_NAMESPACE::PrepopulateBlobCache enum
+  static jbyte toJavaPrepopulateBlobCache(
+      const ROCKSDB_NAMESPACE::PrepopulateBlobCache& prepopulate_blob_cache) {
```

Comment:
It would make sense to pass a small type like this by value as opposed to const-ref.

Reply:
done, thanks